<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>892</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="my_notes_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>23</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Мои записи:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>200</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>12</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 217, 206);</string>
    </property>
    <property name="text">
     <string>Найти запись </string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>200</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 217, 206);</string>
    </property>
    <property name="text">
     <string>Удалить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>660</y>
      <width>200</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 217, 206);</string>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>200</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 217, 206);</string>
    </property>
    <property name="text">
     <string>Изменить запись</string>
    </property>
   </widget>
   <widget class="QTableView" name="record_tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>600</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 217, 206);</string>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="update_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>обновить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
